src/commands/account.ts

```
@@ -117,6 +117,7 @@ export class AccountCommand extends BaseCommand {
         const privateKey = PrivateKey.fromStringDer(newAccountInfo.privateKey);
         newAccountInfo.privateKeyRaw = privateKey.toStringRaw();
       } catch {
+        // TODO should this error be thrown or just logged?
```

probably should throw an error, if it is unable to correctly convert the key, then I don't see what good it will be if we return it, probably will just throw an error in a different location.